### PR TITLE
Ktlint upgrade lint fix

### PR DESCRIPTION
**Pull request by vinitamurthi**
## Explanation
The new Ktlint (version 0.37) is expecting imports to be formatted alphabetically but it will group java and javax imports together.
So you would have normal imports (alphabetically), then imports starting with java (alphabetically), and then imports starting with javax (alphabetically). Right now our linters are failing because we did it alphabetically without any grouping. Creating this PR to demonstrate what the new change would be and get your thoughts on whether we should go forward with 
this?

## Checklist

- [ ] The PR title starts with "Fix #bugnum: ", followed by a short, clear summary of the changes. (If this PR fixes part of an issue, prefix the title with "Fix part of #bugnum: ...".)
- [ ] The PR explanation includes the words "Fixes #bugnum: ..." (or "Fixes part of #bugnum" if the PR only partially fixes an issue).
- [ ] The PR follows the [style guide](https://github.com/oppia/oppia-android/wiki/Coding-style-guide).
- [ ] The PR does not contain any unnecessary auto-generated code from Android Studio.
- [ ] The PR is made from a branch that's **not** called "develop".
- [ ] The PR is made from a branch that is up-to-date with "develop".
- [ ] The PR's branch is based on "develop" and not on any other branch.
- [ ] The PR is **assigned** to an appropriate reviewer in both the **Assignees** and the **Reviewers** sections.

### CODE REVIEW

**Review by BenHenning — APPROVED** [review]
Thanks @vinitamurthi!

File: utility/src/test/java/org/oppia/util/data/DataProvidersTest.kt [hunk 1]
```diff
@@ -2320,7 +2322,8 @@ class DataProvidersTest {
     val baseProvider1 = createSuccessfulDataProvider(BASE_PROVIDER_ID_0, STR_VALUE_0)
     val baseProvider2 = createSuccessfulDataProvider(BASE_PROVIDER_ID_1, STR_VALUE_1)
     val dataProvider = dataProviders.combineAsync<String, String, String>(
-      COMBINED_PROVIDER_ID, baseProvider1, baseProvider2) { _, _ ->
+      COMBINED_PROVIDER_ID, baseProvider1, baseProvider2
```
Comment (rt4914) [inline utility/src/test/java/org/oppia/util/data/DataProvidersTest.kt:2325]: @vinitamurthi Here I think the earlier format made more sense to me.

**Review by rt4914 — COMMENTED** [review]
I have left one comment. 

Also, regarding imports, I am fine with this formatting as earlier also it was like this only. It would be nice to have everything alphabetical but if thats too much work, this will also work easily.

File: utility/src/test/java/org/oppia/util/data/DataProvidersTest.kt (hunk 1 above)
Comment (vinitamurthi) [reply utility/src/test/java/org/oppia/util/data/DataProvidersTest.kt:2325]: Oh? Hmm @anandwana001 do you know what the new ktlint rule is that is enforcing this new line one? We may have to disable it individually

Comment (anandwana001) [reply utility/src/test/java/org/oppia/util/data/DataProvidersTest.kt:2325]: @vinitamurthi is it `disabled_rules=no-blank-line-before-rbrace`?

**Review by anandwana001 — APPROVED** [review]
Seems it worked.

File: utility/src/test/java/org/oppia/util/data/DataProvidersTest.kt (hunk 1 above)
Comment (vinitamurthi) [reply utility/src/test/java/org/oppia/util/data/DataProvidersTest.kt:2325]: Yes that seems to be the one. But @rt4914 from https://github.com/oppia/oppia-android/pull/1234/files#r434257078 it seems like we do want these in new lines. I dont think ktlint behaves differently for `}` or ')'
Given that, shouldn't we keep this?

Comment (rt4914) [reply utility/src/test/java/org/oppia/util/data/DataProvidersTest.kt:2325]: Okay, that makes sense. Then in that case we cannot change this. Got it. thanks.

**Review by rt4914 — APPROVED** [review]

**Pull request merged**